class CacheGetException(Exception):
    """Исключение при получении значения из кэша"""


class CacheSetException(Exception):
    """Исключение при записи значения в кэш"""
